hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/Constants.java

```diff
@@ -33,9 +33,6 @@
   /** Temporary directory. */
   public static final String BUFFER_TMP_KEY = "hadoop.tmp.dir";
 
-  /** Page size for Ozone listing operation. */
-  public static final int LISTING_PAGE_SIZE = 1024;
```

Comment:
Since this is public constant in a public class, I think we should deprecate rather than remove it.  Not sure why anyone would use it downstream, but it's better to retain compatibility.